spring_const_graph.py

```diff
@@ -37,8 +32,10 @@ def calculate_spring_constants(masses, elongations):
     # Calculate spring constants
     test_values = map(spring_constant, masses, elongations)
     spring_constants = list(test_values)
-
-    return spring_constants
+    
+    avg = np.average(spring_constants)*10**(-3)
+    output = print("The spring constant of a given spring is", '{:.2e}'.format(float(avg)), "N/m")
```

Comment:
A print statement as a return value is not a good way to do it. What do you expect it to do?

Reply:
> A print statement as a return value is not a good way to do it. What do you expect it to do?

To show the result without much work in our main.py file. I better change this too and add that line to the main.py.